src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs

```diff
@@ -161,7 +160,9 @@ void StartBackgroundThread()
 		{
 			backgroundThread = new HandlerThread("CameraBackground");
 			backgroundThread.Start();
+#pragma warning disable CS8604 // Possible null reference argument.
 			backgroundHandler = new Handler(backgroundThread.Looper);
```

Comment:
```suggestion
			backgroundHandler = new Handler(backgroundThread!.Looper);
```

Reply:
Yea, there's a lot of strange things like this that we have to work around on Android.

If I'm being frank, the Xamarin.Android team didn't do a great job of implementing Nullability.

I try to avoid `!.` because it could come back to bite us in the future. 

I'll change this to `backgroundThread.Looper ?? throw NullReferenceException()`

Reply:
Sounds great!